docs/setup_details.md

```diff
@@ -1,63 +1,149 @@
+```
+
+#### chocolatey.config
+
+Chocolatey will install multiple packages at the same time when given a config file:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="git" />
+  <package id="nodejs-lts" />
+  <package id="vscode" />
+  <package id="pyenv-win" />
+  <package id="visualstudio2022community" />
+  <package id="visualstudio2022-workload-nativedesktop" />
+  <package id="electron" />
+</packages>
+```
+
+Running `choco install chocolatey.config` is the equivalent of calling `choco install <id>` for every package in the file.
+
+### Python on Windows (pyenv)
+
+Installing and managing Python on Windows is a notoriously difficult task even for advanced developers. There are several one-time steps to set up your system that will drastically reduce future headaches.
+
+First and foremost, turn off both Python App Installers in the "App Execution Aliases" settings. **This is a highly recommend step even if you already have a Python manager**
 
-- Node.js:
-  Install [Node.js](https://nodejs.org/en/download/)
+![App execution aliases](assets/python-win-1.png)
+![Turn off app installer](assets/python-win-2.png)
 
-- Git 2.20.0 or later (with support for "--show-current"):
-  Install [git](https://git-scm.com/downloads/)
+Now we can set up our python management system. We recommend `pyenv` as it is small and intuitive for newer developers. Complete the following steps to install `pyenv` for Windows and setup your global python environment<sup>1</sup>:
 
-- Python 3.7 or later (with support for TLS 1.2):
-  Follow the guide on electron docs to [install and configure python and its modules](https://www.electronjs.org/docs/development/build-instructions-macos#python)
+1) Ensure Python is completely uninstalled from your system
+  
+   - Type `python` on the command line and ensure you get a "Command 'python' not found" error
 
-#### Linux
+     ```powershell
+     python
+     ```
 
-- Node.js:
-  Dowload [Node.js source code](https://nodejs.org/en/download/) and compile it:
+   - _If the command executes, locate the installation and remove it. Continue until the error appears_
+   - _If the Microsoft Store launches check again to ensure "Manage App execution aliases" is turned off for `python.exe` and `python3.exe`_
+
+2) Install `pyenv-win`
+
+   - `pyenv-win` will install with `chocolatey.config` - enter `pyenv` on the command line to see if it's already on your system. You should see a list of commands printed to your terminal.
+
+     ```shell
+     pyenv
+     ```
+
+   - If not:
+
+      ```shell
+      choco install pyenv-win
+      ```
+
+3) Install a python version
+
+   ```powershell
+   pyenv install <version>
+   ```
+
+   - `3.9.6` is a safe starting point if you're unsure of the version you need
```

Comment:
This line will quickly get outdated. Maybe add a date, also, why is that a safe version?

Reply:
Just kind of picked it to be honest, I added instructions for installing the latest version